src/Symfony/Component/Notifier/Bridge/MessageMedia/MessageMediaOptions.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MessageMedia;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author gnito-org <https://github.com/gnito-org>
+ */
+final class MessageMediaOptions implements MessageOptionsInterface
+{
+    private array $options;
+
+    public function __construct(array $options = [])
+    {
+        $this->options = $options;
+    }
+
+    public function getCallbackUrl(): ?string
```

Comment:
Can't we remove all getter? I don't think it's worth exposing them. toArray() works fine for the purpose.
Can you consider this for all your PRs please?

Reply:
Removing all the getters is inconsistent with the [MessageOptionsInterface](https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/Notifier/Message/MessageOptionsInterface.php).